.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
 matrix:
   include:
     - php: '7.0'
-    - php: '7.1'
+    - php: '7.1.0' # Prevent phpdbg's segfaults
```

Comment:
really? if this works I'll really be conviced that there is a "dunglas magic way of fixing stuff"